content/providers/01-ai-sdk-providers/02-openai.mdx

```diff
@@ -899,10 +899,23 @@ const model = openai.image('dall-e-3');
 
 ### Model Capabilities
 
-| Model      | Sizes                           |
-| ---------- | ------------------------------- |
-| `dall-e-3` | 1024x1024, 1792x1024, 1024x1792 |
-| `dall-e-2` | 256x256, 512x512, 1024x1024     |
+| Model         | Sizes                           |
+| ------------- | ------------------------------- |
+| `gpt-image-1` | 1024x1024, 1536x1024, 1024x1536 |
+| `dall-e-3`    | 1024x1024, 1792x1024, 1024x1792 |
+| `dall-e-2`    | 256x256, 512x512, 1024x1024     |
+
+You can pass optional `providerOptions` to the image model. These are prone to change by OpenAI and aremodel dependent. For example, the `gpt-image-1` model supports the `quality` option:
+
+```ts
+const model = openai.image('gpt-image-1', {
+  providerOptions: {
```

Comment:
is this how you pass them? i thought you had to call a method cause this is settings? (either way I still want the ability to do this on all models, think it's handy)

Reply:
This works and is the way AFAIK, though here we don't have a schema defined with the known set of options, which we are moving to, I think we just haven't done the image models yet. Even then, not all options are supported for all models, so it is a world with some imperfections in any case.